### ISSUE TITLE
Separate shift creation/scheduling into its own permission

### ISSUE THREAD

**Issue by nickthenewbie1** [magfest/ubersystem#1473]
We're starting to reach a point where we'd like to people to be able to create and assign shifts for people, but do not want to allow them to create new attendees/staffers.

**Comment by bds002** [comment]
The specific section in Uber is called: Registration and Staffing under the Admin accounts.
https://prime.uber.magfest.org/uber/accounts/

Registration DH's and Staff Ops DH's should be the only people that can go in and arbitrarily create/edit an attendee's contact data, badge status etc.  Department heads should be able to see this data, but not edit it. 

DH should be able to access the following sections of an Attendee's Data page:

Staffing : (yes or no)This attendee is volunteering
(yes or no) Trusted ? 

and:
 the "Assigned Depts" section 

With the ability to  leave Admin notes, or notes to review later.

The tabs for Shifts would fall under the Staffing (DH's access as well) 
The tab for History would also be viewable to review previous events and/or issues.

**Comment by binary1230** [comment]
https://jira.magfest.net/browse/INC-161

**Comment by thaeli** [comment]
Like #1474 this is an upstream issue, please keep it in GitHub

**Comment by binary1230** [comment]
yup, copy. see discussions on #1474  about magfest JIRA, and we'll be keeping this issue right here on github

**Comment by bds002** [comment]
@dom this is related the granulation of permissions we where talking about in the office this past week

**Comment by bds002** [comment]
Revisiting this issue since it affects the Research Team and MAGLabs.  The RT wants to limit the numbers of staff in the system for MAGLabs, however each DH currently has to have access to the Registration/Staffing section of Uber:

![capture](https://cloud.githubusercontent.com/assets/8822346/16776870/4ecaa8a6-4835-11e6-9281-a1821c3d9fb8.PNG)

This should be broken up into two sections so that only Registration or Admin's can add entries to the Database. (Staffing would still be able to see the registration entries, mark and evaluate shifts, leave notes for review etc.....)

**Comment by kitsuta** [comment]
Unfortunately this change would require a pretty major overhaul of admin
accounts and permissions.

The way we control people's access to pages is via the page rendering -
either you can view X or Y page or you can't. There's no such thing as, for
example, a "read-only" attendee details page. We MIGHT be able to separate
staffing and registration, but someone with only staffing access would not
be able to view the attendee's page.
